tests/test_enums.py

```diff
@@ -11,3 +11,9 @@ def test_grey_gray():
 def test_gray_gray():
     """Name of ColorInterp.gray is 'gray'"""
     assert enums.ColorInterp.gray.name == "gray"
+
+
+def test_resampling():
+    """Make sure that resampling value are the same."""
+    for v in enums.OverviewResampling:
+        assert v.value == enums.Resampling[v.name].value
```

Comment:
@vincentsarago how about parametrizing instead? So that we can see more than one failure at a time (if it ever happens).

Reply:
I always forget about pytest.parametrize ;-)